internal/featuretests/externalname_test.go

```diff
@@ -27,6 +28,10 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+func stringPtr(s string) *string {
```

Comment:
I've used this in the past for string pointers: https://github.com/kubernetes/utils/blob/6999998975a717e7f5fc2a7476497044cb111854/pointer/pointer.go#L74

Reply:
Nice!